Microsoft.HealthVault.RestApi/Generated/Models/ActionPlan.cs

```diff
@@ -2,11 +2,16 @@
 // Changes may cause incorrect behavior and will be lost if the code is
 // regenerated.
 
-using System.Collections.Generic;
-using Newtonsoft.Json;
-
 namespace Microsoft.HealthVault.RestApi.Generated.Models
 {
+    using Microsoft.HealthVault;
+    using Microsoft.HealthVault.RestApi;
+    using Microsoft.HealthVault.RestApi.Generated;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
```

Comment:
I think I moved the namespaces with resharper on initial checkin which fixed the them. This is the raw output from autorest (https://github.com/Azure/autorest/blob/master/src/generator/AutoRest.CSharp/Templates/Rest/Client/ServiceClientTemplate.cshtml#L13)